Description: add NEWS entries for previous versions
Origin: upstream
Last-Update: 2014-01-07

--- a/NEWS
+++ b/NEWS
@@ -7,3 +7,145 @@
 Build fixes.
 
 "ocrad.texinfo" has been renamed to "ocrad.texi".
+
+==============================================================================
+
+Changes in version 0.22:
+
+Scaling and smoothing are now made before thresholding.
+
+Character recognition has been improved. (D - O, H - N, O - Q, V - Y, merged TT).
+
+New library function OCRAD_set_utf8_format.
+
+Small improvements have been made in the manual and in the man page.
+
+Quote characters in messages have been changed as advised by GNU Coding
+Standards.
+
+"configure" now accepts options with a separate argument.
+
+Configure option "--datadir" has been renamed to "--datarootdir" to
+follow GNU Standards.
+
+The target "install-bin" has been added to the Makefile.
+
+==============================================================================
+
+Changes in version 0.21:
+
+Some internal errors triggered by noisy input have been fixed.
+
+The chapter "OCR Results File" has been added to the manual.
+
+Stdin and stdout are now set in binary mode on MSVC and OS2.
+
+==============================================================================
+
+Changes in version 0.20:
+
+New library functions:
+  OCRAD_scale
+  OCRAD_result_chars_total
+  OCRAD_result_chars_block
+  OCRAD_result_chars_line
+
+==============================================================================
+
+Changes in version 0.19:
+
+A library interface has been added.
+
+The option "--crop" has been replaced with the similar but different
+option "--cut", which can accept coordinates taken from the ORF file.
+
+Recognition of files with a single character and without white space at
+the edges has been fixed.
+
+Lintian warnings in Debian about man pages have been fixed.
+
+==============================================================================
+
+Changes in version 0.18:
+
+The layout analyser is now able to separate blocks of text of arbitrary
+shape as long as they are clearly delimited by white space.
+
+The new option "--quiet" has been added.
+
+The "--layout" option no more accepts an argument.
+
+The "--crop" option now accepts negative coordinates, interpreted as
+relative to the right-bottom corner of the image.
+
+This version recognizes the letter 'a' with ring above.
+
+Recognition of files with a single big character has been fixed.
+
+A bug that didn't write maxval when saving pgm or ppm files has been fixed.
+
+A problem that prevented compilation with GCC 4.3.0 has been fixed.
+
+"make install-info" should now work on Debian and OS X.
+
+The man page is now installed by default.
+
+Arg_parser has been updated to 1.2.
+
+==============================================================================
+
+Changes in version 0.17:
+
+The license has been updated to GPL version 3 or later.
+
+The "--scale" option no more suppresses ORF output.
+
+The removal of thick frames has been improved.
+
+"Textline" now accepts more than one big initial.
+
+The files "configure" and "Makefile.in" have been modified to be more
+GNU-standards compliant.
+
+==============================================================================
+
+Changes in version 0.16:
+
+A new option "--filter" has been added.
+
+The algorithm for vertical space detection (blank lines) has been improved.
+
+Some fixes to the "configure" script has been made.
+
+Two new debug levels has been added.
+
+Character recognition has been improved.
+
+==============================================================================
+
+Changes in version 0.15:
+
+A new argument parser that replaces "getopt_long" has been added.
+
+A bug that prevented compilation with GCC 4.1 has been fixed.
+
+==============================================================================
+
+Changes in version 0.14:
+
+Ocrad is now able to read ppm (color) files.
+
+The internal representation is now a 256-level greymap instead of a
+bitmap.
+
+Automatic and adaptive binarization has been added, allowing the removal
+of colored backgrounds.
+
+A new option `--crop' has been added.
+
+Two new chapters `Image Format Conversion' and `Algorithm' have been
+added to the texinfo file.
+
+Target `check' has been added to Makefile.
+
+Some fixes and small improvements have been made.
